carsus/io/cmfgen/base.py

```diff
@@ -508,13 +528,19 @@ def from_config(cls, ions, atomic_path, priority=10, ionization_energies=False,
 
                 osc_fname = BASE_PATH.joinpath(ion_keys['osc']
                                                 ).as_posix()
+                col_fname = BASE_PATH.joinpath(ion_keys['col']
+                                                ).as_posix()
 
                 data[ion] = {}
+                
                 lvl_parser = CMFGENEnergyLevelsParser(osc_fname)
                 lns_parser = CMFGENOscillatorStrengthsParser(osc_fname)
-                data[ion]['levels'] = lvl_parser.base
-                data[ion]['lines'] = lns_parser.base
+                col_parser = CMFGENCollisionalStrengthsParser(col_fname)
```

Comment:
Should this not only be done when `collisions` is set to true?

Reply:
Done in [51dcf85](https://github.com/tardis-sn/carsus/pull/272/commits/51dcf858b0cf2ef84fc262a73fb9c3f3de5c7db4)